releases/unreleased/support-ecosystems.yml

```diff
@@ -0,0 +1,35 @@
+---
+title: Support Ecosystems
+category: added
+author: Miguel Ángel Fernández <mafesan@bitergia.com>
+issue: 41
+notes: >
+    This feature allows to add, update and delete Ecosystems
+    to Bestiary. 
+
+    Ecosystems are meant to represent an abstract set of projects
+    which may share a common context.
+
+    An Ecosystem has an identifier and a name. Optionally, they may have
+    a title and a description text. There is a new query allowing to get
+    Ecosystems, including an optional filter to look for an specific ID or name. 
+
+    There are three new mutations:
+
+    * `addEcosystem`: Create a new Ecosystem given a name. Title and description
+    can be included too. Note that each name must be unique and it has to 
+    fulfill some basic requirements:
+      * It has to start with an alphanumeric character.
+      * It cannot contain whitespaces nor punctuaction characters, except
+      hyphens (`-`).
+      * Its length is limited by a parameter defined in the settings.
+
+    * `updateEcosystem`: Update an existing Ecosystem given its ID. The fields 
+    which can be configured are the name, the title and the description.
+    Note that:
+      * The name cannot be updated if that name already exists for another
+        Ecosystem.
+      * The name must follow the same rules defined before.
+
+    * `deleteEcosystem`: Delete an existing Ecosystem given its ID.
```

Comment:
For the purpose of the release notes, we can reduce this part and only enumerate the mutations.